### DISCUSSION TITLE
set-fgtfirewallpolicy

### DISCUSSION THREAD

**Discussion by dawolf18480** [FortiPower/PowerFGT#255] [Ideas]
Is it possible to use the cmdlet set-fgtfirewallpolicy to configure a policy with multiple interfaces for either srcinterface or dstinterface?

**Comment by alagoutte** [comment]
Yes, you can use


```powershell
Get-FGTFirewallPolicy -name MyFGTPolicy| Set-FGTFirewallPolicy -srcintf port1, port2... `
